src/CloudFormation/Result/DescribeStacksOutput.php

```diff
@@ -0,0 +1,232 @@
+        $page = $this;
+        while (true) {
+            if ($page->getNextToken()) {
+                $input->setNextToken($page->getNextToken());
+
+                $nextPage = $this->awsClient->DescribeStacks($input);
+                $this->prefetch[spl_object_hash($nextPage)] = $nextPage;
+            } else {
+                $nextPage = null;
+            }
+
+            yield from $page->getStacks(true);
+
+            if (null === $nextPage) {
+                break;
+            }
+
+            unset($this->prefetch[spl_object_hash($nextPage)]);
+            $page = $nextPage;
+        }
+    }
+
+    protected function populateResult(ResponseInterface $response, HttpClientInterface $httpClient): void
+    {
+        $data = new \SimpleXMLElement($response->getContent(false));
+        $data = $data->DescribeStacksResult;
+
+        $this->Stacks = (function (\SimpleXMLElement $xml): array {
+            $items = [];
+            foreach ($xml as $item) {
+                $items[] = new Stack([
+                    'StackId' => ($v = $item->StackId) ? (string) $v : null,
+                    'StackName' => ($v = $item->StackName) ? (string) $v : null,
+                    'ChangeSetId' => ($v = $item->ChangeSetId) ? (string) $v : null,
+                    'Description' => ($v = $item->Description) ? (string) $v : null,
+                    'Parameters' => (function (\SimpleXMLElement $xml): array {
+                        $items = [];
+                        foreach ($xml as $item) {
+                            $items[] = new Parameter([
+                                'ParameterKey' => ($v = $item->ParameterKey) ? (string) $v : null,
+                                'ParameterValue' => ($v = $item->ParameterValue) ? (string) $v : null,
+                                'UsePreviousValue' => ($v = $item->UsePreviousValue) ? 'true' === (string) $v : null,
+                                'ResolvedValue' => ($v = $item->ResolvedValue) ? (string) $v : null,
+                            ]);
+                        }
+
+                        return $items;
+                    })($item->Parameters),
+                    'CreationTime' => ($v = $item->CreationTime) ? new \DateTimeImmutable((string) $v) : null,
+                    'DeletionTime' => ($v = $item->DeletionTime) ? new \DateTimeImmutable((string) $v) : null,
+                    'LastUpdatedTime' => ($v = $item->LastUpdatedTime) ? new \DateTimeImmutable((string) $v) : null,
+                    'RollbackConfiguration' => new RollbackConfiguration([
+                        'RollbackTriggers' => (function (\SimpleXMLElement $xml): array {
+                            $items = [];
+                            foreach ($xml as $item) {
+                                $items[] = new RollbackTrigger([
+                                    'Arn' => ($v = $item->Arn) ? (string) $v : null,
+                                    'Type' => ($v = $item->Type) ? (string) $v : null,
```

Comment:
This is an issue with the code generator. RollbackTrigger must have two parameters (Arn and Type) and they must be string. Using them like this here allows them to be null. 

This is the error psalm is complaining about.

Reply:
Fixed in #185